go/test/endtoend/vtgate/misc_test.go

```diff
@@ -228,9 +230,9 @@ func TestShowTablesWithWhereClause(t *testing.T) {
 	conn, closer := start(t)
 	defer closer()
 
-	utils.AssertMatches(t, conn, "show tables from ks where Tables_in_ks='t6'", `[[VARCHAR("t6")]]`)
+	utils.AssertMatchesAny(t, conn, "show tables from ks where Tables_in_ks='t6'", `[[VARBINARY("t6")]]`, `[[VARCHAR("t6")]]`)
 	utils.Exec(t, conn, "begin")
-	utils.AssertMatches(t, conn, "show tables from ks where Tables_in_ks='t3'", `[[VARCHAR("t3")]]`)
+	utils.AssertMatchesAny(t, conn, "show tables from ks where Tables_in_ks='t3'", `[[VARBINARY("t3")]]`, `[[VARCHAR("t3")]]`)
```

Comment:
nit: we can also use `utils.AssertMatchesNoOrder` in this case

Reply:
That is different since we want to match to once of the inputs, not all of them (unordered).